tricycle_controller/config/tricycle_drive_controller.yaml

```diff
@@ -1,4 +1,4 @@
-tricycle_controller: 
+tricycle_controller:
```

Comment:
I would try to get these incidental formatting changes into a separate commit or PR.